README.md

```diff
@@ -206,29 +206,33 @@ Read [build instructions and common build issues](BUILD_FROM_SOURCE.md).
 
 ## Testing
 
-1. Download the test scenes from this [link](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally.
+1. Run our python data download utility to retrieve the test assets:
+   ```bash
+   python /path/to/habitat-sim/habitat_sim/utils/datasets_download.py --uid habitat_test_scenes --data_path /path/to/habitat-sim/data/
```

Comment:
```suggestion
   python -m habitat_sim.utils.datasets_download --uid habitat_test_scenes --data_path /path/to/habitat-sim/data/
```

This will work with both a local install and a conda install

Reply:
Additionally, maybe use `/path/to/data` or something to signify that nothing needs to get in a `habitat-sim/data` folder?

Reply:
Looks like there is a bunch of `python /path/to/habitat-sim/habitat_sim/utils/datasets_download.py`, all can/should be `python -m habitat_sim.utils.datasets_download`